js/error-messages.js

```diff
@@ -124,6 +124,6 @@ H.errorMessages = {
             "errors/29/readme.md",
             "errors/30/readme.md"
         ],
-        "version": "6.2.0"
```

Comment:
This change is against the "DO NOT EDIT!" rule placed in this file.
https://github.com/highcharts/highcharts/blob/ec1b1667a800dcf9eead053b89debc21cdeb772c/js/error-messages.js#L8

@bre1470 btw. Could you take a look? Running `gulp` updates this file and the file's current content on the remote seems not up-to-date. How and when will be this file updated?